IDE/Espressif/ESP-IDF/examples/wolfssl_test/main/time_helper.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
```

Comment:
I am okay with pragma, but the `#ifndef _TIME_HELPER_H` syntax is more portable.

Reply:
I was not aware of portability issues. Noted and changed.